src/main/java/core/basesyntax/Application.java

```diff
@@ -1,7 +1,48 @@
 package core.basesyntax;
 
+import core.colors.ColorSupplier;
+import java.util.Random;
+    
 public class Application {
+
+    class Ball {
+        private String namecolor;
+        private int number;
+
+        public Ball(int bound, String namecolor) {
+            Random random = new Random();
+            this.number = bound >= 1 && bound <= 100 ? random.nextInt(bound) : random.nextInt(100);
```

Comment:
there is nothing about this in the task, the developer should **implement** the logic, and not **invent** it
Other than, that there should never be any logic inside constructors, and generating the random number is not the responsibility of a ball, but of a `Lottery` itself